### PR TITLE
Use ftp user for ftp auth

### PR DESCRIPTION
**Pull request by ArnisL**
CLA0003-1018 As an FTP user I wish to be connected to the new storage location so that I can download or upload my files.
CLA0003-1020 As an FTP user I wish to be able to use my FTP username as my username to achieve successful login as that its easy for me to use

### CODE REVIEW

File: templates/default/ftp-auth-handler.erb [hunk 1]
```diff
@@ -2,25 +2,28 @@
...
   f.puts "FTP Login attempted"
   f.puts "Subdomain: #{subdomain}"
   f.puts "Email: #{email}"
 end
 
-res = conn.exec_params("SELECT u.encrypted_password, u.id FROM users u LEFT JOIN subdomain_users su ON su.user_id = u.id
-LEFT JOIN subdomains s ON su.subdomain_id = s.id WHERE s.subdomain LIKE $1 AND u.email LIKE $2", [subdomain, email] )
+conn.exec "SET search_path TO #{subdomain}"
+
+res = conn.exec_params("SELECT u.encrypted_password, u.id FROM ftp_users u WHERE u.username LIKE $1", [username])
```
Comment (ToMoCoop) [inline templates/default/ftp-auth-handler.erb]: Should this be a LIKE or an EQ? I think it should equal username?

File: templates/default/ftp-auth-handler.erb [hunk 2]
```diff
@@ -2,25 +2,28 @@
...
+
 open(log_file, 'a') do |f|
   f.puts "FTP Login attempted"
   f.puts "Subdomain: #{subdomain}"
   f.puts "Email: #{email}"
 end
 
-res = conn.exec_params("SELECT u.encrypted_password, u.id FROM users u LEFT JOIN subdomain_users su ON su.user_id = u.id
-LEFT JOIN subdomains s ON su.subdomain_id = s.id WHERE s.subdomain LIKE $1 AND u.email LIKE $2", [subdomain, email] )
+conn.exec "SET search_path TO #{subdomain}"
```
Comment (ToMoCoop) [inline templates/default/ftp-auth-handler.erb]: What happens is the subdomain doesn't exist?

Comment (ArnisL) [reply templates/default/ftp-auth-handler.erb]: will error out or fail at looking up ftp user later on. seems fine to me

File: templates/default/ftp-auth-handler.erb (hunk 1 above)
Comment (ArnisL) [reply templates/default/ftp-auth-handler.erb]: good spot. will change

**Pull request merged**